sycl/include/CL/sycl/detail/kernel_program_cache.hpp

```diff
@@ -25,29 +25,35 @@ namespace detail {
 class context_impl;
 class KernelProgramCache {
 public:
+  struct BuildResultT {
```

Comment:
This sounds like a build error, rather than a build result. The structure doesn't seem to contain any result at all.

Reply:
Done